var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -74,7 +74,13 @@ def url_for_version(self, version):
             return "{0}/v{1}.tar.gz".format(base_url, version)
 
     def autoreconf(self, spec, prefix):
-        which('bash')('autogen.sh')
+        if self.spec.satisfies("@:1.1.6"):
+            libtoolize()
+            aclocal('-I', 'lib/autoconf')
+            autoconf()
+        else:
+            # autogen.sh file will be available from v2 and above
+            which('bash')('autogen.sh')
```

Comment:
Shouldn't the logic be switched here:
```suggestion
        if self.spec.satisfies("@2:"):
            which('bash')('autogen.sh')
        else:
            # Fall back since autogen is not available
            libtoolize()
            aclocal('-I', 'lib/autoconf')
            autoconf()
```

Reply:
Hi @sethrj , thanks for the suggestions, we have addressed them in latest pull request, please do the needful.